### ISSUE TITLE
[FEATURE]: CSS-variable for font-family

### ISSUE THREAD

**Issue by projct1** [jackdomleo7/Cooltipz.css#65]
Now is hardcoded. Have to use crutch:
```css
[aria-label][data-cooltipz-dir]::after, [aria-label][class*=cooltipz]::after {
  font-family: $font-family-base;
}
```

**Comment by jackdomleo7** [comment]
Will investigate this customisation as I can see it conflicting with FontAwesome config.

_Please stick to the issue template provided, it helps prioritise tasks._

**Comment by jackdomleo7** [comment]
Released in [v2.1.0](https://github.com/jackdomleo7/Cooltipz.css/releases/tag/v2.1.0)
